source/includes/indexes/index-code-examples.rb

```diff
@@ -0,0 +1,112 @@
+require 'mongo'
+
+# Replace the placeholders with your credentials
+uri = "<connection string>"
+
+# Sets the server_api field of the options object to Stable API version 1
+options = { server_api: { version: "1" }}
```

Comment:
[q] I noticed the other placeholders in this doc use single quotes, do these need to be in double quotes? I'm actually not sure the preference in Ruby but curious if they should all be single quotes for consistency.

Reply:
I'm not sure what the correct syntax is. It seems to work with single quotes, double quotes, and as a plain integer! Will ask tech reviewer.